
From: Colin Leroy <colin@colino.net>

This one fixes another unitialized var warning (which is harmless, but
whatever).

Signed-off-by: Colin Leroy <colin@colino.net>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/macintosh/macio-adb.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/macintosh/macio-adb.c~warning-fix-in-drivers-macintosh-macio-adbc drivers/macintosh/macio-adb.c
--- 25/drivers/macintosh/macio-adb.c~warning-fix-in-drivers-macintosh-macio-adbc	2004-10-05 11:19:36.227604304 -0700
+++ 25-akpm/drivers/macintosh/macio-adb.c	2004-10-05 11:19:36.230603848 -0700
@@ -202,7 +202,7 @@ static irqreturn_t macio_adb_interrupt(i
 				       struct pt_regs *regs)
 {
 	int i, n, err;
-	struct adb_request *req;
+	struct adb_request *req = NULL;
 	unsigned char ibuf[16];
 	int ibuf_len = 0;
 	int complete = 0;
_
